pkg/volume/csi/csi_client_test.go

```diff
@@ -415,3 +453,37 @@ func TestClientNodeUnstageVolume(t *testing.T) {
 		}
 	}
 }
+
+func TestNodeExpandVolume(t *testing.T) {
+	testCases := []struct {
+		name       string
+		volID      string
+		volumePath string
+		newSize    resource.Quantity
+		mustFail   bool
+		err        error
+	}{
+		{name: "with all correct values", volID: "vol-abcde", volumePath: "/foo/bar", newSize: resource.MustParse("10Gi"), mustFail: false},
```

Comment:
can you format this each variable on a different line to make it easier to read?